src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,62 @@
 databaseChangeLog:
   -  changeSet:
        id:  adicionar suporte ao liquibase
-       author:  izumizawa
\ No newline at end of file
+       author:  izumizawa
+
+  - changeSet:
+      id: add-address-table
+      author: Gabriel Seabra
+      changes:
+        - createTable:
+            tableName: address
+            columns:
+              - column:
+                  name: address_id
+                  type: int
```

Comment:
Is it possible to change this type to UUID, here are some documentation (the example is in xml format, but it can be used in yaml) https://docs.liquibase.com/workflows/liquibase-community/working-with-uuids.html

Also remove the autoincrement if using uuids :)

Reply:
Do I have to create that .csv file? or is it optional in our case?